app.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable no-unused-vars */
+let i = 0;
+let list = [];
+function g(j) {
+  return document.getElementById(j);
+}
+function stack(templateString) {
+  const template = document.createElement('template');
+  template.innerHTML = templateString.trim();
+  document.getElementById('books').appendChild(template.content.firstChild);
+}
+function addBook() {
+  const title = g('t').value;
+  const author = g('a').value;
+  stack(`<div id="${i + title}">
+      <p>${title}</p>
+      <p>${author}</p>
+      <button id="${i + title}" value="${i}" onclick="removeBook(this.id, this.value)">Remove</button>
+    </div>
+    <hr>`);
+  const locaList = JSON.parse(localStorage.getItem('stuff')).arr;
+  if (locaList !== undefined) {
+    list = locaList;
+    const book = {};
+    book.author = author;
+    book.title = title;
+    list.push(book);
+    i += 1;
+    localStorage.setItem('stuff', JSON.stringify({ arr: list }));
+  } else {
+    const book = {};
+    book.author = author;
+    book.title = title;
+    list.push(book);
+    i += 1;
+    localStorage.setItem('stuff', JSON.stringify({ arr: list }));
+  }
+  g('t').value = '';
+  g('a').value = '';
+}
+function removeBook(id, p) {
+  const position = parseInt(p, 10);
+  list.splice(position, 1);
+  document.getElementById(id).remove();
+  localStorage.setItem('stuff', JSON.stringify({ arr: list }));
+}
+/* eslint-disable no-plusplus */
+function loadPrev() {
+  const mList = JSON.parse(localStorage.getItem('stuff')).arr;
```

Comment:
- [ ] As mentioned by the last reviewer, your app does not retain books data when the page is reloaded. It looks like you're having some issues with your localStorage. I tried to run your application locally and I found the following error in my console: 

![shot](https://user-images.githubusercontent.com/26856491/124589476-f3447800-de51-11eb-9d59-3731a9f491f1.png)

Can this be the problem? You might want to check this line and see what's actually causing the error:
https://github.com/softwareengineermwi/lib/blob/aaf1b2ab2f1abb8b8759dcbfe01b05549f65e6a1/app.js#L49